### PR TITLE
[VP] IR expansion for bitreverse/bswap

### PR DESCRIPTION
**Pull request by LiqinWeng**

### CODE REVIEW

File: llvm/lib/CodeGen/ExpandVectorPredication.cpp [hunk 1]
```diff
@@ -702,7 +702,17 @@ Value *CachingVPExpander::expandPredication(VPIntrinsic &VPI) {
...
+  }
+  case Intrinsic::vp_bswap:
+  case Intrinsic::vp_bitreverse: {
+    Value *Op = VPI.getOperand(0);
+    Function *Fn = Intrinsic::getDeclaration(
+        VPI.getModule(), VPI.getFunctionalIntrinsicID().value(),
+        {VPI.getType()});
+    Value *NewOp = Builder.CreateCall(Fn, {Op}, VPI.getName());
+    replaceOperation(*NewOp, VPI);
+    return NewOp;
```
Comment (RKSimon) [inline llvm/lib/CodeGen/ExpandVectorPredication.cpp:715]: Maybe worth moving this to expandPredicationToIntCall and adding unary handling?

Comment (LiqinWeng) [reply llvm/lib/CodeGen/ExpandVectorPredication.cpp]: done

**Review by RKSimon — APPROVED** [review]
LGTM

**Review by lukel97 — APPROVED** [review]
LGTM

**Pull request merged**